Dfe.Academies.External.Web/Services/IConversionApplicationCreationService.cs

```diff
@@ -10,24 +10,7 @@ public interface IConversionApplicationCreationService
 	Task AddSchoolToApplication(int applicationId, int schoolUrn, string name);
 
 	Task AddTrustToApplication(int applicationId, int trustUkPrn, string name);
-
-	Task ApplicationAddJoinTrustReasons(int applicationId, string applicationJoinTrustReason, int schoolUrn);
-
-	Task ApplicationChangeSchoolNameAndReason(int applicationId, SelectOption changeName,
-		string? newSchoolName, int schoolUrn);
-
-	Task ApplicationSchoolTargetConversionDate(int applicationId, int schoolUrn, SelectOption targetDateDifferent, DateTime targetDate, string targetDateExplained);
-
-	Task ApplicationSchoolFuturePupilNumbers(int applicationId, int schoolUrn, int projectedPupilNumbersYear1, int projectedPupilNumbersYear2,
-		int projectedPupilNumbersYear3, string schoolCapacityAssumptions, int schoolCapacityPublishedAdmissionsNumber);
-
-	Task ApplicationSchoolContacts(ApplicationSchoolContacts schoolContacts, int applicationId);
-
-	Task ApplicationSchoolLandAndBuildings(SchoolLandAndBuildings schoolLandAndBuildings, int applicationId, int schoolUrn);
-
-	Task ApplicationPreOpeningSupportGrantUpdate(PayFundsTo schoolSupportGrantFundsPaidTo, bool confirmSupportGrantToSchool, int applicationId, int schoolUrn);
-
-	Task ApplicationSchoolPreviousFinancialYear(SchoolFinancialYear previousFinancialYear, int applicationId, int schoolUrn);
-
+	
+	Task PutSchoolApplicationDetails(int applicationId, int schoolUrn, Dictionary<string, dynamic> schoolProperties);
```

Comment:
I actually hate the idea of the API having one gigantic PUT which is why I didn't want the UI to have one gigantic PUT. Hence why there are separate updates as per each part of the UI / steps in the wizard. Taking a front - to - back design approach